Resources/Prototypes/Corvax/Reagents/Consumable/Drink/alcohol.yml

```
@@ -0,0 +1,292 @@
+- type: reagent
+  id: Campari
+  name: reagent-name-campari
+  parent: BaseAlcohol
+  desc: reagent-desc-campari
+  physicalDesc: reagent-physical-desc-bubbly
+  flavor: alcohol
+  color: "#da2059"
+  metabolisms:
+    Drink:
+      effects:
+      - !type:SatiateThirst
```

yamllint

[error] wrong indentation: expected 8 but found 6

(indentation)